.github/workflows/tag-maintainers.yml

```diff
@@ -0,0 +1,248 @@
+name: Sync Plugin Maintainer Reviewers
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review, reopened, synchronize]
+
+# Concurrency settings to ensure that only one instance of this workflow runs per PR.
+# If a new commit is pushed, it cancels the previous run.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read       # To checkout code
+  pull-requests: write # To add/remove reviewers and comment
+
+jobs:
+  tag-maintainers:
+    runs-on: ubuntu-latest
+    if: |
+      github.event.pull_request.draft == false &&
+      github.event.pull_request.state == 'open'
+
+    steps:
+      # Step 1: Generate a GitHub App token if configured.
+      - name: Create GitHub App token
+        uses: actions/create-github-app-token@v2
+        if: vars.CI_APP_ID
+        id: app-token
+        with:
+          app-id: ${{ vars.CI_APP_ID }}
+          private-key: ${{ secrets.CI_APP_PRIVATE_KEY }}
+          permission-contents: write
+          permission-pull-requests: write
+          permission-members: read
+
+      # Step 2: Checkout the code from the base branch.
+      # This is a security measure for `pull_request_target` to run trusted code.
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+
+      # Step 3: Get the specific Nixpkgs revision from the lock file.
+      - name: Get nixpkgs
+        id: nixpkgs
+        run: |
+          jq --raw-output --exit-status '
+            .nodes[.nodes[.root].inputs.nixpkgs].locked
+            | "https://github.com/\(.owner)/\(.repo)/archive/\(.rev).tar.gz"
+            | "url=\(.)"
+          ' flake.lock >> "$GITHUB_OUTPUT"
+
+      # Step 4: Install Nix with the correct configuration.
+      - name: Install Nix
+        uses: cachix/install-nix-action@v31
+        with:
+          nix_path: nixpkgs=https://github.com/NixOS/nixpkgs/archive/${{ steps.get-nixpkgs.outputs.rev }}.tar.gz
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+
+      # Step 5: Identify which plugin files have changed in the PR.
+      - name: Get changed plugin files
+        id: changed-files
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token || secrets.GITHUB_TOKEN }}
+          PR_NUM: ${{ github.event.pull_request.number }}
+        run: |
```

Comment:
Just for my own understanding: `gh pr diff` will show the pr diff against its base (as it would appear on the website), **not** the diff against the local worktree (`git diff HEAD..FETCH_HEAD`)?

Reply:
I'm not 100% certain of the differences, I was just using `gh` cli for the other commands and continued the trend.